app/views/content/roadmap.md

```diff
@@ -1,52 +1,47 @@
 <p class="govuk-body-l">Find out about recent and upcoming service changes.</p>
 
-<p class="govuk-hint">Last updated: 26 January 2022</p>
+<p class="govuk-hint">Last updated: 9 May 2022</p>
 
 ***
 
 ### Changes to services for providers
 
 Recent changes include:
 
-* introducing a [report to enable providers to view the status of active applications](https://bat-design-history.netlify.app/manage-teacher-training-applications/helping-users-check-how-quickly-courses-are-filling-up/)
-
-* making it easier for providers to [check what qualifications candidates have](https://bat-design-history.netlify.app/manage-teacher-training-applications/improving-the-qualifications-section-of-the-application-details-page/) by improving how applications are laid out
-
 * trialling online chat to enable providers to contact Department for Education (DfE) support agents quickly
 
 * publishing a [report giving statistics for teacher training applications](https://www.apply-for-teacher-training.service.gov.uk/publications/monthly-statistics), which will be updated monthly
 
+* [making it easier for providers to find and track applications](https://bat-design-history.netlify.app/manage-teacher-training-applications/finding-out-the-effect-of-removing-the-reference-number/) by improving the use of reference numbers
+
+* making it easier for [providers to give candidates feedback](https://bat-design-history.netlify.app/manage-teacher-training-applications/reasons-for-rejection-iteration-6/) when their application was unsuccessful
+
+* releasing [version 1.1 of the API](https://www.apply-for-teacher-training.service.gov.uk/api-docs/draft), which lets users manage applications in their own student record systems - the new version will let users schedule interviews, make notes, defer applications and withdraw applications at a candidate‘s request
```

Comment:
will this URL be updated on/after the 9th?

Reply:
Good question. How about I change it now to https://www.apply-for-teacher-training.service.gov.uk/api-docs/v1.1/reference ?